File: Python/2475. Number of Unequal Triplets in Array.py
class Solution:
    def unequalTriplets(self, nums: List[int]) -> int:
        res, l = 0, len(nums)

        for i in range(l):
            for j in range(i + 1, l):
                for k in range(j + 1, l):
                    if nums[i] != nums[j] and nums[j] != nums[k] and nums[i] != nums[k] and i != k:
                        res += 1
        
        return res

File: Java/2475. Number of Unequal Triplets in Array.java
class Solution {
    public int unequalTriplets(int[] nums) {
        int ans = 0, l = nums.length;

        for(int i = 0; i < l; i++){
            for(int j = i + 1; j < l; j++){
                for(int k = j + 1; k < l; k++){
                    if(nums[i] != nums[j] && nums[j] != nums[k] && nums[i] != nums[k]){
                        ans++;
                    }
                }
            }
        }

        return ans;
    }
}
